modules/contact-form/grunion-contact-form.php

```diff
@@ -2038,13 +2038,36 @@ function process_submission() {
 		 * with this work around. */
 		add_filter( 'wp_insert_post_data', array( $plugin, 'insert_feedback_filter' ), 10, 2 );
 
+		$disabled_functions = explode( ',', ini_get( 'disable_functions' ) );
+		
+		// Avoid warnings when print_r() has been disabled on hosting provider for security reasons
+		if ( ! in_array( 'print_r', $disabled_functions ) ) {
+			$all_values_string = print_r( $all_values, true );
+		}
+		// If print_r is disabled, try to use var_export as an alternative solution
+		elseif ( ! in_array( 'var_export', $disabled_functions ) ) {
+			$all_values_string = var_export( $all_values, true );
+		}
+		else {
+			$all_values_string = '';
+		}
+		
+		$post_content_string =  $comment_content . "\n<!--more-->\n" . "AUTHOR: {$comment_author}\n
+		AUTHOR EMAIL: {$comment_author_email}\n
```

Comment:
The prior string used to insert just a `\n` between lines, the new code does that and then also does an actual new line (I've not examined it closely enough to identify if it's `\r\n` or just another `\n`) followed by tabs.  This would change it so that the formatting is not consistent with the prior entries.  :\

While it is more readable in code, it's less reliable for any code trying to parse out the content (not the best way to do things, but the things I've seen ...)